src/main/java/com/internet/shop/controller/product/AddProductController.java

```diff
@@ -0,0 +1,34 @@
+package com.internet.shop.controller.product;
+
+import com.internet.shop.lib.Injector;
+import com.internet.shop.model.Product;
+import com.internet.shop.service.ProductService;
+import java.io.IOException;
+import java.math.BigDecimal;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/addProduct")
+public class AddProductController extends HttpServlet {
+    private static final Injector INJECTOR = Injector.getInstance("com.internet.shop");
+    private final ProductService productService = (ProductService) INJECTOR
+            .getInstance(ProductService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException,
+            IOException {
+        req.getRequestDispatcher("/WEB-INF/view/product/addProduct.jsp").forward(req, resp);
+
+    }
```

Comment:
```suggestion
        req.getRequestDispatcher("/WEB-INF/view/product/addProduct.jsp").forward(req, resp);
    }
```
будь ласка, прибері всі зайві пусті рядки у класах